components/gateway/cmd/main.go

```diff
@@ -71,7 +73,7 @@ func main() {
 
 	http.Handle("/", router)
 
-	log.Printf("Listening on %s", cfg.Address)
+	log.Printf("Listening on %s\n", cfg.Address)
```

Comment:
The new line in the end is not needed. The log package adds new line for each print